build/templates/UmbracoProject/.gitignore

```diff
@@ -0,0 +1,490 @@
+*.binlog
+
+# NVidia Nsight GPU debugger configuration file
+*.nvuser
+
+# MFractors (Xamarin productivity tool) working folder
+.mfractor/
+
+# Local History for Visual Studio
+.localhistory/
+
+# BeatPulse healthcheck temp database
+healthchecksdb
+
+# Backup folder for Package Reference Convert tool in Visual Studio 2017
+MigrationBackup/
+
+# Ionide (cross platform F# VS Code tools) working folder
+.ionide/
+
+# Fody - auto-generated XML schema
+FodyWeavers.xsd
+
+##
+## Visual studio for Mac
+##
+
+
+# globs
+Makefile.in
+*.userprefs
+*.usertasks
+config.make
+config.status
+aclocal.m4
+install-sh
+autom4te.cache/
+*.tar.gz
+tarballs/
+test-results/
+
+# Mac bundle stuff
+*.dmg
+*.app
+
+# content below from: https://github.com/github/gitignore/blob/master/Global/macOS.gitignore
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
+
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+# content below from: https://github.com/github/gitignore/blob/master/Global/Windows.gitignore
+# Windows thumbnail cache files
+Thumbs.db
+ehthumbs.db
+ehthumbs_vista.db
+
+# Dump file
+*.stackdump
+
+# Folder config file
+[Dd]esktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Windows Installer files
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# JetBrains Rider
+.idea/
+*.sln.iml
+
+##
+## Visual Studio Code
+##
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+
+
+##
+## Umbraco CMS .NETCore
+##
+
+# Dont commit Umbraco TEMP folder containing Examine Indexes, NuCache etc
+**/umbraco/Data/TEMP/
+
+# Dont commit files that are generated and cached from the default ImageSharp location
+**/umbraco/mediacache/
+
+# Umbraco backoffice language files
+# Nuget package Umbraco.Cms.StaticAssets will copy them in during dotnet build
+# Customize langguage files in /config/lang/{language}.user.xml
+**/umbraco/config/lang/
+
+# JSON Schema file for appsettings
+# This is auto generated from the build
+**/umbraco/config/appsettings-schema.json
+
+# This is the no-nodes, installer & upgrader pages from Umbraco
+# Nuget package Umbraco.Cms.StaticAssets will copy them in during dotnet build
+**/umbraco/UmbracoWebsite/
+**/umbraco/UmbracoInstall/
+**/umbraco/UmbracoBackOffice/
+
+# Comment out the line below if you wish to change or add any new templates to PartialView Macros
+**/umbraco/PartialViewMacros/Templates
+
+# Umbraco Static Assets of Backoffice
+# Nuget package Umbraco.Cms.StaticAssets will copy them in during dotnet build
+**/wwwroot/umbraco/assets/
+**/wwwroot/umbraco/js/
+**/wwwroot/umbraco/lib/
+**/wwwroot/umbraco/views/
```

Comment:
```suggestion
**/wwwroot/umbraco/
```

Reply:
Is there any other folders in `wwwroot/umbraco/` that I may want to commit?

Reply:
There should not be that..

Reply:
Ok will change this part then